apps/minifront/src/fetchers/registry.ts

```diff
@@ -34,6 +35,20 @@ export const getStakingTokenMetadata = async () => {
   return stakingAssetsMetadata;
 };
 
+export const getAssetTokenMetadata = async (assetId: AssetId) => {
```

Comment:
will need to be modified after https://github.com/penumbra-zone/web/pull/1472 is merged